internal/k8s/kind.go

```diff
@@ -45,6 +46,8 @@ func KindOf(obj interface{}) string {
 			return "HTTPProxy"
 		case *gatewayapi_v1beta1.HTTPRoute:
 			return "HTTPRoute"
+		case *gatewayapi_v1alpha2.GRPCRoute:
+			return "GRPCRoute"
```

Comment:
Will have to look to see exactly where this is used; I'm noticing we don't have TLSRoute here.

Reply:
Right, TLSRoute is not there. For routes, KindOf() is used in cache.go, statusaddress.go and metrics.go for some debug/error message or metric counter.

Reply:
:+1: thanks, yeah makes sense to add GRPCRoute here, we should follow up with adding any other missing types for now, and/or see if we can implement this functionality differently such that we don't need to maintain a hardcoded list.